readme.md

```diff
@@ -21,7 +21,7 @@ Includes:
 - Reactotron (requires 2.x)
 - And more!
 
-To see it in action, check out these [examples](https://github.com/infinitered/ignite-bowser-examples)!
+To see it in action, check out the Chain React 2019 Conference App [example](https://github.com/infinitered/ChainReactApp2019)!
```

Comment:
Super small nit: would this read better if "Chain React 2019 Conference App" was the text of the link, instead of "example"?

Reply:
@robinheinze - I believe it would! I've updated it as suggested.